document/CoreFormat.mdk

```diff
@@ -1082,33 +1173,18 @@ Whether `analysisTarget` is present depends on the information available to the
 ### `resultFile` property { #location-resultFile }
 
 A `location` object may contain a property named `resultFile` whose value is a `physicalLocation` object (&sect;[#physicalLocation])
-that identifies the file where the static analysis tool produced the result.
+that identifies the file where the analysis tool produced the result.
 
 Whether `resultFile` is present depends on the information available to the tool that produces the log file
 (see &sect;[#location-Constraints]).
 
-### `logicalLocation` property { #location-logicalLocation }
-
-A `location` object may contain a property named `logicalLocation`,
-which specifies the logical location where the static analysis tool detected the result.
-
-A logical location specifies a programmatic construct, for example, a class name or a function name,
-without specifying the programming artifact within which that construct occurs.
-
-The value of the `logicalLocation` property shall be
-an array of `logicalLocationComponent` objects (&sect;[#logicalLocationComponent]).
-This allows reference to results which occur in nested logical components, for example,
-a method nested within a class which is in turn nested within a namespace.
-When a result occurs in such a nested construct, the first `logicalLocationComponent` object in the array shall refer to
-the outermost construct,
-the next element shall refer to a construct nested immediately within the first construct,
-and so on, with the last element referring to the most deeply nested construct.
-
 ### `fullyQualifiedLogicalName` property { #location-fullyQualifiedLogicalName }
 
 A `location` object may contain a property named `fullyQualifiedLogicalName` whose value is a string
-which summarizes the information in the `logicalLocationComponent` array (&sect;[#logicalLocationComponent])
-in a format consistent with the programming language in which the programming constructs specified by that array were expressed.
+which specifies the fully qualified name of the logical location where the analysis tool produced the result.
+
+The format of the `fullyQualifiedLogicalName` string shall be consistent with the programming language
+in which the programmatic construct specified by that logical location was expressed.
 
```

Comment:
here's where i expected to find more useful information about the fully qualified logical name. It is a readable name that uniquely identifies the item under analysis. It is therefore highly suited to display to users to help identify the analysis target in question. As a readable item that uniquely identifies something, it is useful for fingerprinting (including in-source suppressions where, again, the readability is an advantage).